connectivity/lwipstack/source/LWIPStack.cpp

```diff
@@ -687,6 +766,19 @@ nsapi_error_t LWIP::setsockopt(nsapi_socket_t handle, int level, int optname, co
             }
             s->conn->pcb.ip->tos = (u8_t)(*(const int *)optval);
             return 0;
+
+        case NSAPI_PKTINFO:
+            // FIXME: Turn NETCONN_FLAG_PKTINFO off by default
+            if (optlen != sizeof(int)) {
+                return NSAPI_ERROR_UNSUPPORTED;
```

Comment:
Can you address the not and set `NETCONN_FLAG_PKTINFO` off when the socket is opened ?

Reply:
Done